src/stats/hexbin.jl

```diff
@@ -26,6 +26,10 @@ function spacings_offsets_nbins(bins::Tuple{Int,Int}, cellsize::Nothing, xmi, xm
     return xspacing, yspacing, xmi, ymi, bins...
 end
 
+# hardcoded scale factors
+xfact() = 2
+yfact() = 4 / 3
```

Comment:
Can these be `hexbin_xfact` or underscore methods?  People doing `Makie.x<tab>` will likely see this, might get a bit confusing...

Reply:
Thanks, you're right, done in https://github.com/MakieOrg/Makie.jl/pull/4898/commits/6355b3a6d6c97c68f140c95a2b30aea2aa019e9f.